moveit_planners/ompl/ompl_interface/src/model_based_planning_context.cpp

```diff
@@ -795,7 +795,7 @@ bool ompl_interface::ModelBasedPlanningContext::solve(double timeout, unsigned i
 
       ob::PlannerTerminationCondition ptc = constructPlannerTerminationCondition(timeout, start);
       registerTerminationCondition(ptc);
-      result = ompl_parallel_plan_.solve(ptc, 1, count, hybridize_) == ompl::base::PlannerStatus::EXACT_SOLUTION;
+      result.val = errorCode(ompl_parallel_plan_.solve(ptc, 1, count, hybridize_));
```

Comment:
Well here you changed the results by converting `APPROXIMATE` to `true` instead of `false` as before

Reply:
Should be addressed by the latest commit